### PR TITLE
(docs) Fix comment spelling mistakes

### PR DESCRIPTION
**Pull request by l0b0**

### CODE REVIEW

File: lib/puppet/pops/loader/module_loaders.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@ module Loader
 # internal layout etc.)
 #
 # A module loader is also not aware of the mapping of name to relative paths - this is performed by the
-# included module PathBasedInstantatorConfig which knows about the map from type/name to
+# included module PathBasedInstantiatorConfig which knows about the map from type/name to
```
Comment (Iristyle) [inline lib/puppet/pops/loader/module_loaders.rb]: It looks like this never actually got implemented per https://github.com/l0b0/puppet/commit/96f3d9720b3dc0289689a090a21855ec5bc09310 @hlindberg ?

File: lib/puppet/pops/resource/param.rb [hunk 2]
```diff
@@ -15,7 +15,7 @@
 module Puppet::Pops
 module Resource
 class Param
-  # This make this class instantiateable from Puppet
+  # This make this class instantiatable from Puppet
```
[the remark this answers is not in the document]
Comment (l0b0) [reply lib/puppet/pops/resource/param.rb]: Better than [my reference](https://en.wiktionary.org/wiki/instantiatable) :) - will fix.

File: lib/puppet/util/docs.rb [hunk 3]
```diff
@@ -96,7 +96,7 @@ def markdown_definitionlist(term, definition)
   # formatting of the code they're embedded in. Since indents are syntactic
   # elements in Markdown, we need to make sure we remove any indent that was
   # added solely to preserve surrounding code formatting, but LEAVE any indent
-  # that delineates a Markdown element (code blocks, multi-line bulleted list
```
Comment (jtappa) [inline lib/puppet/util/docs.rb]: this was not wrong

Comment (ericldelaney) [reply lib/puppet/util/docs.rb]: @l0b0 can you change this one back to "bulleted" please

File: lib/puppet/pops/loader/module_loaders.rb (hunk 1 above)
Comment (hlindberg) [reply lib/puppet/pops/loader/module_loaders.rb]: loaders were refactored - that logic got changed along the way. Suggest updating the comment to just say: "A module loader is also not aware of the mapping of name to relative paths." and leave out any mention of how it is implemented.

Comment (ericldelaney) [reply lib/puppet/pops/loader/module_loaders.rb]: @l0b0 Can you make the suggested change?

**Pull request merged**